provisioner/ibm-s3fs-provisioner.go

```diff
@@ -637,6 +645,22 @@ func (p *IBMS3fsProvisioner) Provision(ctx context.Context, options controller.P
 			}
 		}
 
+		if pvc.BucketVersioning != "" {
+			enable := pvc.BucketVersioning == "true"
+			if out, err := sess.SetBucketVersioning(pvc.Bucket, enable); err != nil {
+				if err != nil {
+					if deleteBucket {
+						err1 := sess.DeleteBucket(pvc.Bucket)
+						if err1 != nil {
+							return nil, controller.ProvisioningFinished, fmt.Errorf(pvcName+" : "+clusterID+" :cannot set bucket versioning %v", err1, " and cannot delete bucket %s :  %v", pvc.Bucket, err)
+						}
+					}
+					return nil, controller.ProvisioningFinished, fmt.Errorf(pvcName+":"+clusterID+" :failed to set version for bucket %s: %v", pvc.Bucket, err)
```

Comment:
Use 
`return nil, controller.ProvisioningFinished, fmt.Errorf(pvcName+":"+clusterID+" :failed to set versioning %s for bucket %s: %v", enable, pvc.Bucket, err)`